### DISCUSSION TITLE
Good News - Net Code Partially Works

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#25] [Announcements]
After a full week of deducing what I wrote, I managed to get something to function in the mod again, and we have a tornado finally!

![2022-01-21_15 30 48](https://user-images.githubusercontent.com/50820746/150613396-7516b5c6-4b9a-497c-947a-457ecb065a87.png)

While it only sits there and does nothing (Because some methods are still under work), it is definitely progress. Now that I know that my net code is finally being done correctly, I can now start getting things to move and work again.

I'll update you guys shortly when I know things work fully again.

**Comment by zad112** [comment]
NICE
